source/tests/unit/ni_fake_service_tests.cpp

```diff
@@ -498,6 +498,33 @@ TEST(NiFakeServiceTests, NiFakeService_AcceptListOfDurationsInSeconds_CallsAccep
   EXPECT_EQ(kDriverSuccess, response.status());
 }
 
+TEST(NiFakeServiceTests, NiFakeService_BoolArrayOutputFunction_CallsBoolArrayOutputFunction)
```

Comment:
I see this one for `ViBoolean[]` outputs. Could you write one for boolean array inputs?

Reply:
I didn't find any function in niFake which accepts ViBoolean[] input parameter. Do I need to create one?

 I am leaving this comment open and completing the pull request to unblock nidcpower example Palash is working on which depends on FetchMultiple API. Based on outcome of this discussion I will submit test for input array as separate pull request.

Reply:
Yes, please create one. You'll need to mention that in the CHANGES.md file. It'll eventually need to get backported to the hapigen metadata in perforce, so we should have the change documented.

Reply:
Just to close loop here, I have created [PR for ViBoolean array inputs](https://github.com/doshirohan/grpc-device/pull/41). This should go to owners review shortly.